tests/transforms/test_add_noise.py

```diff
@@ -221,7 +221,7 @@ def test_add_noise_dev(self, dev_name):
         new_program, _ = new_dev.preprocess()
         [tape], _ = new_program([in_tape])
         res_with_noise = qml.execute(
-            [in_tape], new_dev, qml.gradients, transform_program=new_program
+            [in_tape], new_dev, qml.gradients.param_shift, transform_program=new_program
```

Comment:
I think this supports my desire to mode gradient method validation inside `qml.execute`... 😬  How did that even work 🫠

Reply:
I know 🫨 !